src/domain/app/i18n/fi.json

```
@@ -1883,6 +1882,39 @@
       "placeholderConfirmationMessage": "Syötä vahvistusviesti ilmoittautuneille",
       "placeholderEvent": "Valitse tai hae tapahtuma",
       "placeholderInstructions": "Syötä ilmoittautumisohjeet",
+      "registrationAccount": {
+        "buttonReturnDefaults": "Palauta tilin oletusarvot",
+        "helperDefaultValue": "Oletusarvo: {{value}}",
+        "labelAccount": "Kirjanpitotili",
+        "labelBalanceProfitCenter": "Tasapainotettu tulosyksikkö",
+        "labelCompanyCode": "SAP-yrityskoodi",
```

```suggestion
        "labelCompanyCode": "SAP-yritystunnus",
```
Term used in Jarkko's PPTX.

---

Changed as suggested